<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="ru.rearitem.ServerTest" tests="1" skipped="0" failures="1" errors="0" timestamp="2018-08-24T08:35:12" hostname="WS-GALUZIN" time="0.005">
  <properties/>
  <testcase name="test1" classname="ru.rearitem.ServerTest" time="0.005">
    <failure message="java.util.NoSuchElementException: No line found" type="java.util.NoSuchElementException">java.util.NoSuchElementException: No line found
	at java.util.Scanner.nextLine(Scanner.java:1540)
	at ru.rearitem.ServerTest.test1(ServerTest.java:69)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.runTestClass(JUnitTestClassExecutor.java:106)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:58)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:38)
	at org.gradle.api.internal.tasks.testing.junit.AbstractJUnitTestClassProcessor.processTestClass(AbstractJUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:109)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:155)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:137)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:745)
</failure>
  </testcase>
  <system-out><![CDATA[2018-08-24 11:35:12 DEBUG HikariConfig:919 - HikariPool-1 - configuration:
2018-08-24 11:35:12 DEBUG HikariConfig:948 - allowPoolSuspension.............false
2018-08-24 11:35:12 DEBUG HikariConfig:948 - autoCommit......................false
2018-08-24 11:35:12 DEBUG HikariConfig:948 - catalog.........................none
2018-08-24 11:35:12 DEBUG HikariConfig:948 - connectionInitSql...............none
2018-08-24 11:35:12 DEBUG HikariConfig:948 - connectionTestQuery.............none
2018-08-24 11:35:12 DEBUG HikariConfig:948 - connectionTimeout...............30000
2018-08-24 11:35:12 DEBUG HikariConfig:948 - dataSource......................none
2018-08-24 11:35:12 DEBUG HikariConfig:948 - dataSourceClassName.............none
2018-08-24 11:35:12 DEBUG HikariConfig:948 - dataSourceJNDI..................none
2018-08-24 11:35:12 DEBUG HikariConfig:948 - dataSourceProperties............{password=<masked>, prepStmtCacheSqlLimit=2048, cachePrepStmts=true, prepStmtCacheSize=250}
2018-08-24 11:35:12 DEBUG HikariConfig:948 - driverClassName.................none
2018-08-24 11:35:12 DEBUG HikariConfig:948 - healthCheckProperties...........{}
2018-08-24 11:35:12 DEBUG HikariConfig:948 - healthCheckRegistry.............none
2018-08-24 11:35:12 DEBUG HikariConfig:948 - idleTimeout.....................600000
2018-08-24 11:35:12 DEBUG HikariConfig:948 - initializationFailFast..........true
2018-08-24 11:35:12 DEBUG HikariConfig:948 - initializationFailTimeout.......1
2018-08-24 11:35:12 DEBUG HikariConfig:948 - isolateInternalQueries..........false
2018-08-24 11:35:12 DEBUG HikariConfig:948 - jdbc4ConnectionTest.............false
2018-08-24 11:35:12 DEBUG HikariConfig:948 - jdbcUrl........................."jdbc:h2:mem:test;MODE=PostgreSQL;DB_CLOSE_DELAY=-1;INIT=runscript from 'classpath:/db.sql'"
2018-08-24 11:35:12 DEBUG HikariConfig:948 - leakDetectionThreshold..........0
2018-08-24 11:35:12 DEBUG HikariConfig:948 - maxLifetime.....................1800000
2018-08-24 11:35:12 DEBUG HikariConfig:948 - maximumPoolSize.................10
2018-08-24 11:35:12 DEBUG HikariConfig:948 - metricRegistry..................none
2018-08-24 11:35:12 DEBUG HikariConfig:948 - metricsTrackerFactory...........none
2018-08-24 11:35:12 DEBUG HikariConfig:948 - minimumIdle.....................10
2018-08-24 11:35:12 DEBUG HikariConfig:948 - password........................<masked>
2018-08-24 11:35:12 DEBUG HikariConfig:948 - poolName........................"HikariPool-1"
2018-08-24 11:35:12 DEBUG HikariConfig:948 - readOnly........................false
2018-08-24 11:35:12 DEBUG HikariConfig:948 - registerMbeans..................false
2018-08-24 11:35:12 DEBUG HikariConfig:948 - scheduledExecutor...............none
2018-08-24 11:35:12 DEBUG HikariConfig:948 - scheduledExecutorService........internal
2018-08-24 11:35:12 DEBUG HikariConfig:948 - threadFactory...................internal
2018-08-24 11:35:12 DEBUG HikariConfig:948 - transactionIsolation............default
2018-08-24 11:35:12 DEBUG HikariConfig:948 - username........................"sa"
2018-08-24 11:35:12 DEBUG HikariConfig:948 - validationTimeout...............5000
2018-08-24 11:35:12 INFO  HikariDataSource:71 - HikariPool-1 - Starting...
2018-08-24 11:35:12 DEBUG HikariPool:519 - HikariPool-1 - Added connection conn0: url=jdbc:h2:mem:test user=SA
2018-08-24 11:35:12 INFO  HikariDataSource:73 - HikariPool-1 - Start completed.
2018-08-24 11:35:12 INFO  ServerTest:31 - get conn HikariProxyConnection@151268339 wrapping conn0: url=jdbc:h2:mem:test user=SA
2018-08-24 11:35:12 DEBUG logging:59 - Logging Provider: org.jboss.logging.Log4jLoggerProvider
2018-08-24 11:35:12 DEBUG session:384 - Setting default session timeout to 1800
2018-08-24 11:35:12 DEBUG HikariPool:390 - HikariPool-1 - Pool stats (total=1, active=0, idle=1, waiting=0)
2018-08-24 11:35:12 DEBUG PoolBase:370 - HikariPool-1 - Failed to create/setup connection: Таблица "ACCOUNTS" уже существует
Table "ACCOUNTS" already exists; SQL statement:
create table accounts(
    account_uid varchar not null primary key,
    account_name varchar not null,
    account_email varchar unique not null,
    account_pass bytea not null
    ) [42101-195]
2018-08-24 11:35:12 DEBUG HikariPool:464 - HikariPool-1 - Cannot acquire connection from data source
org.h2.jdbc.JdbcSQLException: Таблица "ACCOUNTS" уже существует
Table "ACCOUNTS" already exists; SQL statement:
create table accounts(
    account_uid varchar not null primary key,
    account_name varchar not null,
    account_email varchar unique not null,
    account_pass bytea not null
    ) [42101-195]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:117)
	at org.h2.command.dml.RunScriptCommand.execute(RunScriptCommand.java:78)
	at org.h2.command.dml.RunScriptCommand.update(RunScriptCommand.java:57)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:224)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:354)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:100)
	at org.h2.Driver.connect(Driver.java:69)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:95)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:101)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:356)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:199)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:444)
	at com.zaxxer.hikari.pool.HikariPool.access$200(HikariPool.java:71)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:631)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:617)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-24 11:35:12 DEBUG session:224 - Registered session listener io.undertow.servlet.core.SessionListenerBridge@402eda49
2018-08-24 11:35:12 DEBUG undertow:113 - starting undertow server io.undertow.Undertow@4d3a358e
2018-08-24 11:35:12 INFO  xnio:99 - XNIO version 3.3.6.Final
2018-08-24 11:35:12 INFO  nio:55 - XNIO NIO Implementation Version 3.3.6.Final
2018-08-24 11:35:12 DEBUG HikariPool:464 - HikariPool-1 - Cannot acquire connection from data source
org.h2.jdbc.JdbcSQLException: Таблица "ACCOUNTS" уже существует
Table "ACCOUNTS" already exists; SQL statement:
create table accounts(
    account_uid varchar not null primary key,
    account_name varchar not null,
    account_email varchar unique not null,
    account_pass bytea not null
    ) [42101-195]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:117)
	at org.h2.command.dml.RunScriptCommand.execute(RunScriptCommand.java:78)
	at org.h2.command.dml.RunScriptCommand.update(RunScriptCommand.java:57)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:224)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:354)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:100)
	at org.h2.Driver.connect(Driver.java:69)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:95)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:101)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:356)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:199)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:444)
	at com.zaxxer.hikari.pool.HikariPool.access$200(HikariPool.java:71)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:631)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:617)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-24 11:35:12 DEBUG nio:431 - Started channel thread 'XNIO-1 I/O-1', selector sun.nio.ch.WindowsSelectorImpl@72193d36
2018-08-24 11:35:12 DEBUG nio:431 - Started channel thread 'XNIO-1 I/O-2', selector sun.nio.ch.WindowsSelectorImpl@791295a
2018-08-24 11:35:12 DEBUG nio:431 - Started channel thread 'XNIO-1 I/O-3', selector sun.nio.ch.WindowsSelectorImpl@65236090
2018-08-24 11:35:12 DEBUG nio:431 - Started channel thread 'XNIO-1 I/O-6', selector sun.nio.ch.WindowsSelectorImpl@72814480
2018-08-24 11:35:12 DEBUG nio:431 - Started channel thread 'XNIO-1 Accept', selector sun.nio.ch.WindowsSelectorImpl@26ddc6f3
2018-08-24 11:35:12 DEBUG nio:431 - Started channel thread 'XNIO-1 I/O-8', selector sun.nio.ch.WindowsSelectorImpl@799f0e67
2018-08-24 11:35:12 DEBUG undertow:150 - Configuring listener with protocol HTTP for interface localhost and port 58088
2018-08-24 11:35:12 DEBUG nio:431 - Started channel thread 'XNIO-1 I/O-4', selector sun.nio.ch.WindowsSelectorImpl@f3c11c5
2018-08-24 11:35:12 DEBUG nio:431 - Started channel thread 'XNIO-1 I/O-5', selector sun.nio.ch.WindowsSelectorImpl@5996d8e4
2018-08-24 11:35:12 DEBUG nio:431 - Started channel thread 'XNIO-1 I/O-7', selector sun.nio.ch.WindowsSelectorImpl@76f5f292
2018-08-24 11:35:12 DEBUG undertow:219 - stopping undertow server io.undertow.Undertow@4d3a358e
2018-08-24 11:35:12 INFO  HikariDataSource:347 - HikariPool-1 - Shutdown initiated...
2018-08-24 11:35:12 DEBUG HikariPool:390 - HikariPool-1 - Before shutdown stats (total=1, active=0, idle=1, waiting=0)
2018-08-24 11:35:12 DEBUG PoolBase:134 - HikariPool-1 - Closing connection conn0: url=jdbc:h2:mem:test user=SA: (connection evicted)
2018-08-24 11:35:13 DEBUG HikariPool:390 - HikariPool-1 - After shutdown stats (total=0, active=0, idle=0, waiting=0)
2018-08-24 11:35:13 INFO  HikariDataSource:349 - HikariPool-1 - Shutdown completed.
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
